eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -77,13 +57,35 @@ function DeployStressTests(
     [string]$searchDirectory = '.',
     [hashtable]$filters = @{},
     [string]$environment = 'test',
-    [string]$repository = 'images',
+    [string]$repository = '',
     [boolean]$pushImages = $false,
-    [string]$clusterGroup = 'rg-stress-cluster-test',
```

Comment:
Do we not want this as default?
I thought this does apply to most cases when we run this script?

Reply:
so far there's only test and prod env. situaations that overrides this value, will there be other cases where we need clusterGroup specified?

Reply:
It gets defaulted below now, but I changed the defaulting behavior to be based on the value of environment.

Reply:
Will there be any cases that it might accidentally fall through the if/elif?

Reply:
They would fall through, but then fail. Perhaps I should add a specific throw here to handle it still being empty after the checks.